subsys/mgmt/mcumgr/smp_udp.c

```diff
@@ -65,27 +65,52 @@ static struct configs configs = {
 static int smp_udp4_tx(struct zephyr_smp_transport *zst, struct net_buf *nb)
 {
 	ARG_UNUSED(zst);
+	int ret;
+
+	if (CONFIG_MCUMGR_SMP_UDP_MTU == 0U) {
```

Comment:
Here in `udp6` version, same as in uart.

Reply:
Fixed